tests/end2end-tests/data_process/gen_test_graphs.py

```diff
@@ -0,0 +1,374 @@
+"""
```

Comment:
this script is not used in CI. I wonder if we need to have such a file checked in the test folder.

Reply:
we already have a script that generates a test graph. https://github.com/awslabs/graphstorm/blob/main/tests/end2end-tests/data_process/data_gen.py
we run the pipeline on the test graph and verify the correctness of the partition result.

Reply:
This script will be used for regression test.